src/StackAdmin/Dns/Commands.Dns/Models/DoubleFetchHandler.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
@deathly809 please add the license header

Reply:
The file has headers

Reply:
No it doesn't... lol.  Let me do a PR to add missing headers.